### DISCUSSION TITLE
Editor setting to hide array "size" in inspector

### DISCUSSION THREAD

**Discussion by pixel-boy** [godotengine/godot-proposals#11148] [Editor]
![array size](https://github.com/user-attachments/assets/9bbf8c07-d162-45be-abc7-437981196d60)
I use exported arrays a lot but never use this line and it makes the readability of the array more difficult, especially when they are used in multiple sub-resources.

I guess it's used for some cases so adding an option to hide it in the editor settings could be good I think.

**Comment by zmn-hamid** [comment]
I really need this one. where you have too many nested resources things start to look real messy because of this exact field. unfortunately i found no addon for it. hopefully i can make one myself

**Comment by Calinou** [comment]
https://github.com/godotengine/godot/pull/103257 partially implements this proposal by moving the Size property to the bottom, where it's generally less of a disturbance. You can still control its position with an editor setting.
